pkg/skaffold/event/v2/logger.go

```diff
@@ -87,8 +81,8 @@ func (h logHook) Levels() []logrus.Level {
 // Fire constructs a SkaffoldLogEvent and sends it to the event channel
 func (h logHook) Fire(entry *logrus.Entry) error {
 	handler.handleSkaffoldLogEvent(&proto.SkaffoldLogEvent{
-		TaskId:    fmt.Sprintf("%s-%d", h.task, handler.iteration),
-		SubtaskId: h.subtask,
+		TaskId:    fmt.Sprintf("%s-%d", entry.Data["task"], handler.iteration),
```

Comment:
Can you verify if this will not cause a null pointer exception

Reply:
(if the key does not exists)

Reply:
my bad, just pushed changes to fix this